### PR TITLE
Paym 106 metrics

### PR DESCRIPTION
**Pull request by mniesyn**
https://lodgify.atlassian.net/browse/PAYM-106

The metrics was added in style:
- company naming (prefix as microservice name etc)
- good practices https://prometheus.io/docs/practices/naming/

Maybe for now we have little bit too much metrics because there was created for cases: creating, created, failed but then we can see everything like (please reach the /metrics uri) :

payments_stripe_create_account_session_total{result="creating"} 3
payments_stripe_create_account_session_total{result="created"} 1
payments_stripe_create_account_session_total{result="failed"} 2

After contact with Isaac and explain how metrics are consume from application I know it is not possible to build grafana dashboard with data from local env, it is impossible because locally there is no mechanism for actively pulling metrics by prometheus. It must be separate task.

### CODE REVIEW

File: Lodgify.Payments.Stripe.Application.UnitTests/UseCases/CreateAccount/CreateAccountCommandHandlerTests.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ﻿using FluentAssertions;
+using Lodgify.Architecture.Metrics.Abstractions;
```
Comment (marcinmilik) [inline Lodgify.Payments.Stripe.Application.UnitTests/UseCases/CreateAccount/CreateAccountCommandHandlerTests.cs:2]: Is this using needed?
```suggestion
```

File: Lodgify.Payments.Stripe.Application/Metrics/Metrics.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lodgify.Payments.Stripe.Application.Metrics;
+
+public static class Metrics
+{
+     public const string AccountCreated = "lodgify_payments_stripe_new_account_created";
+     public const string AccountSessionCreated = "lodgify_payments_stripe_new_account_session_created";
+}
```
Comment (marcinmilik) [inline Lodgify.Payments.Stripe.Application/Metrics/Metrics.cs:7]: I cannot find where these consts are used. Do we use them?

**Review by marcinmilik — COMMENTED** [review]
>  I know it is not possible to build grafana dashboard with data from local env

Could you please create a separate task to create grafana dashboards once we deploy our service to live? We can set is as SELECTED FOR DEVELOPMENT as mark as blocked by PAYM-110.
How does it sound?

File: Lodgify.Payments.Stripe.Application/Metrics/Metrics.cs (hunk 2 above)
Comment (mniesyn) [reply Lodgify.Payments.Stripe.Application/Metrics/Metrics.cs:7]: Ahh no, it was before I was decided to make separate project for metric stuff

**Review by marcinmilik — APPROVED** [review]

**Pull request merged**